charts/prometheus-adapter/values.yaml

```
@@ -254,6 +254,12 @@ podAnnotations: {}
 # Annotations added to the deployment
 deploymentAnnotations: {}
 
+# Annotations added to the pod security policy
+pspAnnotations: {}
```

Nest the annotations under the existing `psp` name.

```suggestion
psp:
  annotations: {}
```